assets/dev/scss/editor/panel/_elements.scss

```diff
@@ -25,20 +18,20 @@
 	padding: 0 5px;
 }
 
-#elementor-panel-elements-loading {
-	position: absolute;
-	top: 0;
-	left: 0;
-	height: 100%;
-	width: 100%;
-	display: flex;
-	align-items: center;
-	justify-content: center;
-	background-color: rgba(255, 255, 255, 0.5);
-	z-index: 1;
+#elementor-panel-state-loading {
+	display: none;
 
-	body.elementor-controls-ready & {
-		display: none;
+	body.elementor-panel-loading & {
```

Comment:
What happened here?

Reply:
The "Loading" state is now not only for controls. it works according to the body class `elementor-panel-loading`